### DISCUSSION TITLE
XML | ignore-ssl

### DISCUSSION THREAD

**Discussion by madmann26** [grafana/grafana-infinity-datasource#406] [Q&A]
Hi,

I have some servers with self-signed certs (I know, fix the certs). When I attempt to connect to them, I am seeing -

curl: (60) SSL certificate problem: unable to get local issuer certificate

Is it possible to pass the "-k" variable?

Thanks,

**Comment by yesoreyeram** [comment]
Check the datasource configuration page for ignoring server cert errors

**Reply by yesoreyeram** [reply]
@madmann26 did that help?
